spec/requests/user_spec.rb

```diff
@@ -0,0 +1,107 @@
+require "gds_api/test_helpers/content_store"
+
+RSpec.describe "User information endpoint" do
+  include GdsApi::TestHelpers::ContentStore
+
+  before do
+    stub_oidc_discovery
+
+    attributes.each do |name, value|
+      if value.nil?
+        stub_request(:get, "http://openid-provider/v1/attributes/#{name}").to_return(status: 404)
+      else
+        stub_request(:get, "http://openid-provider/v1/attributes/#{name}").to_return(body: { claim_value: value }.to_json)
+      end
+    end
```

Comment:
Same as the previous comment: can we put this in a `stub_requests_for_attributes(attributes)` method or similar?
Mainly to abstract some of the complexity away from here.